cmd/soroban-cli/src/invoke.rs

```diff
@@ -49,6 +50,9 @@ pub struct Cmd {
     /// Argument to pass to the function (base64-encoded xdr)
     #[clap(long = "arg-xdr", value_name = "arg-xdr", multiple = true)]
     args_xdr: Vec<String>,
+    /// File containing JSON Argument to pass to the function
+    #[clap(long, multiple = true)]
```

Comment:
Are args files XDR files? That wasn't obvious to me on first read, I thought it was JSON still. Maybe we should make the option `--arg-xdr-file` to make it clear.
```suggestion
    #[clap(long, value_name = "arg-xdr-file", multiple = true)]
```

Separately we could have `arg-xdr` for the JSON form?

Reply:
Yeah I meant for this one to be the same as `--args`, which is both json and raw strings right?  The one test in this repo used `--args=world`.  I'll go ahead and add a `args-xdr-file` to ensure that there is clarity.